west.yml

```diff
@@ -176,6 +178,12 @@ manifest:
       repo-path: sdk-openthread
       path: modules/lib/openthread
       revision: 0d19f9112101e87722ec80b3a247bc7a1c54b232
+    - name: ant
+      repo-path: sdk-ant
+      revision: 5cd2bce18f707c3132415536d539c934a4e1a3ee
+      remote: ant-nrfconnect
```

Comment:
For a single repository I think it would be cleaner to do

```
- name: ant
  url: https://github.com/ant-nrfconnect/sdk-ant
  ...
```

But this is fine too.

Reply:
I kept the remote for now... fine to change in future if you'd like to enforce the pattern for single repos.